HHS/Head-Start-TTADP
# [TTAHUB-3336] Extract PR number for deployment completion slack notification.

## Description of change
Adds logic to process merge commits to extract the PR number for the deployment completion slack notifications.


## How to test
- Verify that the sample message on deployment to sandbox in the acf-head-start-github channel looks good.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3336


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Before merge to main

- [n/a] OHS demo complete

### Production Deploy


### After merge/deploy
